README.md

```diff
@@ -148,6 +150,62 @@ Example:
 
 ![Set Kedro project path](assets/kedro-project-path.gif)
 
+## Auto reload Kedro-Viz
```

Comment:
Should we add some gifs for easier visualisation of these instructions?

Reply:
100%, pls let's add gifs. I mentioned in the other PR.